### ISSUE TITLE
Translation keys part of comment must not be valid and must be reported as an error/warning

### ISSUE THREAD

**Issue by sebd71** [svoboda-rabstvo/ngx-translate-lint#110]
# Bug report

## Describe the bug

Translation keys part that are part of comment in typescript or in html are currently considered as used.

## To Reproduce

- Declare a key in en.json language file and add a reference of this key in comment either in ts or in html :
```ts
// this is my comment that refer the key STRING.KEY_COMMENT
```
or
```html
<!--   STRING.KEY_COMMENT -->
```

## Expected behavior

Translation keys part that are inside a comment section must be considered as not used and must raise an error (or warning).
It is maybe possible to update regexp of findKeysList function (file keys.ts) to detect typescript comment (line beginning with `//`) but comment detection in html is more tricky because comment may defined on several lines.
